biz.aQute.tester/src/aQute/junit/Activator.java

```diff
@@ -736,6 +742,21 @@ private void message(String prefix, Throwable t, String string) {
 		}
 	}
 
+	@SuppressWarnings("serial")
+	private void terminate(int exitcode) {
+		if (terminate) {
+			System.exit(exitcode);
+		}
+		else {
+			context.registerService(IntSupplier.class, () -> 0, new Hashtable<String, Object>() {
```

Comment:
This is a terrible construct (anonymously extending to add the value in the constructor). Please don't do this.